components/Nav/NavListItem/NavListItem.js

```diff
@@ -96,6 +105,9 @@ export default class NavListItem extends Component {
                       role="link"
                       tabIndex={0}
                       data-testid={`Nav Item ${sublink.name}`}
+                      onKeyDown={
```

Comment:
Can you put all of the logic into `handleKeyDown`  instead of some here and some in the method definition?

So just: `event => this.handleKeyDown(event, index);`